torchbenchmark/models/BERT_pytorch/bert_pytorch/model/attention/single.py

```diff
@@ -16,7 +16,11 @@ def forward(self, query, key, value, dropout: TensorToTensor, mask: Optional[tor
                  / math.sqrt(query.size(-1))
 
         if mask is not None:
-            scores = scores.masked_fill(mask == 0, -1e9)
+            if scores.dtype == torch.float16:
+                min_mask = -65504.0
+            else:
+                min_mask = -1e9
+            scores = scores.masked_fill(mask == 0, min_mask)
```

Comment:
This line is filling the `score` variable with `min_mask` for values that are `0`. Does this change the semantics of the code? Looks like it is.

Reply:
Generally, it won't change the model representation as the min_mask is used as mask to the empty token. So using the minimum float is what the paper suggested.

Reply:
> Generally, it won't change the model representation as the min_mask is used as mask to the empty token. So using the minimum float is what the paper suggested.

Can you add comments to the code with a link to the paper so that people understand this change?